BackEnd/src/controllers/githubController.js

```diff
@@ -163,8 +163,301 @@ const getPullRequests = async (req, res) => {
   }
 }
 
+/**
+ * PR 상세 정보 조회 (본문, 코멘트, README 포함)
+ * GET /api/github/pull/:owner/:repo/:number/details
+ * 헤더: X-GitHub-Token: <GitHub Access Token>
+ */
+const getPullRequestDetails = async (req, res) => {
+  try {
+    console.log('=== getPullRequestDetails 함수 진입 ===')
+    console.log('요청 경로:', req.path)
+    console.log('요청 파라미터:', req.params)
+
+    if (!req.user) {
+      console.log('사용자 정보 없음 - authMiddleware 실패')
+      return sendUnauthorized(res, 'Not authenticated')
+    }
+
+    console.log('사용자 인증 완료:', req.user.id)
+
+    const { owner, repo, number } = req.params
+    const accessToken = req.headers['x-github-token']
+
+    console.log('파라미터 확인:')
+    console.log('- owner:', owner)
+    console.log('- repo:', repo)
+    console.log('- number:', number)
+    console.log('- GitHub Token:', accessToken ? '있음' : '없음')
+
+    if (!owner || !repo || !number) {
```

Comment:
이경우에 꼭 적용할 필요는 없지만, if안의 조건문이 복잡하다면 변수명을 붙여두는 식으로 좀더 유지보수가 쉽게 만들어줄 수 있어요.
```suggestion
    const isMissingParams = !owner || !repo || !number
    if (isMissingParams) {
```
조건의 목적을 설명해주는 변수명을 지어주면 코드를 한눈에 파악하기 더쉽습니다 👍